### PR TITLE
42crunch url

### PR DESCRIPTION
**Pull request by jasmingacic**
This PR returns Crunch42URL in the APIItem. 
In order to achieve that I added KGW dependency from the main branch hence `github.com/kubeshop/kusk-gateway v1.5.1-0.20221207083051-c3507a110867` 

Also you will notice I didn't handle any errors for 42Crunch as we don't want to break if we can't get the URL

Unblocks https://github.com/kubeshop/kusk-gateway/issues/974

Signed-off-by: jasmingacic <jasmin.gacic@gmail.com>

### CODE REVIEW

File: server/go/model_api_item.go [hunk 1]
```diff
@@ -19,6 +19,8 @@ type ApiItem struct {
 	Service ApiItemService `json:"service"`
 
 	Version string `json:"version"`
+
+	Crunch42URL string `json:"crunc42url"`
```
Comment (mbana) [inline server/go/model_api_item.go]: Mis-spelling: Shouldn't it be `crunch42url` or rather `42crunch_url`?

File: api/openapi.yaml [hunk 2]
```diff
@@ -690,6 +690,8 @@ components:
           $ref: "#/components/schemas/ApiItem_Service"
         version:
           type: string
+        crunch42URL:
```
Comment (mbana) [inline api/openapi.yaml:693]: In another file, you've got it spelt as `crunc42url`. I'd use `42crunch_URL` or `42crunch_url`. Whatever you prefer.

**Review by mbana — CHANGES_REQUESTED** [review]
LGTM for the most part. Just a typo I found.

File: api/openapi.yaml (hunk 2 above)
Comment (jasmingacic) [reply api/openapi.yaml:693]: I fixed the typo. I'm not a fan of using underscores for variable namees.

Comment (mbana) [reply api/openapi.yaml:693]: You have

```go
Crunch42URL string `json:"crunch42url"`
```

in `server/go/model_api_item.go`.

and in `api/openapi.yaml` you have `crunch42URL`.

Does the casing make a difference?

Comment (jasmingacic) [reply api/openapi.yaml:693]: oh :D

File: server/go/model_api_item.go (hunk 1 above)
Comment (jasmingacic) [reply server/go/model_api_item.go]: fixed it

**Review by mbana — DISMISSED** [review]
LGTM.

I'm not too sure about the update to `github.com/kubeshop/kusk-gateway` to `v1.5.1-0.20221207083051-c3507a110867`.

**Review by mbana — DISMISSED** [review]
LGTM.

**Pull request merged**